### PR TITLE
Add WX staking adapter

### PR DESCRIPTION
**Pull request by vba2000**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
wx network

##### Website Link:
https://wx.network/

##### Current TVL:
WX staking 1.65M

##### Treasury Addresses (if the protocol has treasury)
WX Staking contract 3PJL8Hn8LACaSBWLQ3UVhctA5cTQLBFwBAP

##### Chain:
WAVES

##### Short Description (to be shown on DefiLlama):
WX staking

##### Token address and ticker if any:
Token Id in WAVES Atqv59EYzjFGuitKVnMRk6H8FukjoV3ktPorbEys25on ticker WX

##### Category (full list at https://defillama.com/categories) *Please choose only one:
10

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL of WX means the quantity of staked WX tokens on Staking contract

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/wx-network-wx exports TVL: 
        
 
 
    waves                     1.64 M
    
    total                    1.64 M

File: projects/wx-network-wx/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  const { balance } = await assetBalance(WXStakingWxTokenContract, WXAssetId);
+  sdk.util.sumSingleBalance(balances,'waves-exchange', balance / 1e8);
+  return balances;
+}
+
+module.exports = {
+  timetravel: false, // Waves blockchain
+  methodology: "TVL of WX means the quantity of staked WX tokens on WX Staking contract",
+  waves: {
+    tvl: WXTVL,
```
Comment (Define101) [inline projects/wx-network-wx/index.js:18]: staking , since its your own token
We dont count own tokens as TVL, should be staking

Comment (vba2000) [reply projects/wx-network-wx/index.js:18]: It's users tokens in stake
https://wx.network/governance/wx_staking

Comment (Define101) [reply projects/wx-network-wx/index.js:18]: But is your governance token , so it belong to staking and not TVL 
image
https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Comment (vba2000) [reply projects/wx-network-wx/index.js:18]: Ok. Thank you.

**Pull request closed without merging**